### PR TITLE
fix(dockerbuild): architectures

### PR DESCRIPTION
**Pull request by rluckom-coil**
fixes [#1263](https://github.com/interledger/rafiki/issues/1263)

Also uses base images specific to the architectures which seems to speed up the builds a lot

## Changes proposed in this pull request

- 

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [ ] Documentation added
- [ ] Make sure that all checks pass

### CODE REVIEW

**Comment by sabineschaller** [comment]
While you are working on this, could you include this?
- #1263

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -13,22 +13,14 @@ jobs:
...
     steps:
       - name: Docker meta
         id: meta
         uses: docker/metadata-action@v4
         with:
           images: ghcr.io/${{ github.repository_owner }}/rafiki-${{ matrix.package }}
-          tags: |
-            type=raw,enable={{is_default_branch}},value=latest
-            type=raw,enable=true,value={{ sha }}
-            type=raw,enable=true,value={{ refs }}
```
Comment (sabineschaller) [inline .github/workflows/docker.yml:31]: Does the removal of this ensure that the image tag =  gh tag?

**Review by sabineschaller — COMMENTED** [review]
Docker images work for me. 

And I'm assuming that the docker image tagging with the gh tag also works since I see a new tag in the list of tags?

**Comment by sabineschaller** [comment]
Also, can you add the 
- fixes #1263 

to your description if it is actually working?

File: .github/workflows/docker.yml (hunk 1 above)
Comment (rluckom-coil) [reply .github/workflows/docker.yml:31]: yeah, the default behavior is to add the git tag as a label and the other defaults do what we were already doing (branch name, commit sha, etc)

**Review by sabineschaller — APPROVED** [review]
Thank you!

**Pull request merged**